# Source: https://gitlab.com/gitlab-org/gitlab/-/blob/v18.11.7/Gemfile
# CVE-2025-67202 requires sidekiq-cron 2.4.0: https://nvd.nist.gov/vuln/detail/CVE-2025-67202
# GHSA-3h96-34p3-xm76 requires graphql 2.5.26: https://github.com/advisories/GHSA-3h96-34p3-xm76
# sidekiq-cron 2.4 compatibility: https://github.com/sidekiq-cron/sidekiq-cron/compare/v2.3.1...v2.4.0
diff --git a/Gemfile b/Gemfile
--- a/Gemfile
+++ b/Gemfile
@@ -169 +169 @@
-gem 'graphql', '2.5.11', feature_category: :api
+gem 'graphql', '~> 2.5.26', feature_category: :api
@@ -299,3 +299,3 @@
 gem 'sidekiq', path: 'vendor/gems/sidekiq', require: 'sidekiq', feature_category: :scalability
-gem 'sidekiq-cron', '~> 2.3.0', feature_category: :scalability
+gem 'sidekiq-cron', '~> 2.4.0', feature_category: :scalability
 gem 'gitlab-sidekiq-fetcher',
diff --git a/lib/gitlab/patch/sidekiq_cron_poller.rb b/lib/gitlab/patch/sidekiq_cron_poller.rb
--- a/lib/gitlab/patch/sidekiq_cron_poller.rb
+++ b/lib/gitlab/patch/sidekiq_cron_poller.rb
@@ -13 +13 @@
-if Gem::Version.new(Sidekiq::Cron::VERSION) != Gem::Version.new('2.3.1')
+if Gem::Version.new(Sidekiq::Cron::VERSION) != Gem::Version.new('2.4.0')
